dipy/viz/colormap.py

```diff
@@ -255,3 +261,52 @@ def line_colors(streamlines, cmap='rgb_standard'):
                     for streamline in streamlines]
 
     return np.vstack(col_list)
+
+
+lowercase_cm_name = {'blues': 'Blues', 'accent': 'Accent'}
+
+
+def create_colormap(v, name='jet', auto=True):
```

Comment:
Nope there is no reason to change the default behavior of this function neither to add another warning. What we need to do is use different colormaps in our tutorials. I will add something in the documentation. To suggest using viridis or inferno. Sounds good? By the way I don't think there is one colormap that is good for all the different jobs.